### ISSUE TITLE
Floating point number precision problem

### ISSUE THREAD

**Issue by phyng** [ijl/orjson#12]
OS: Ubuntu 16.04
orjson Version: 2.0.1

```python
In [8]: data = {'lat': 31.245270191439438, 'lng': 121.48791951161945}

In [9]: orjson.loads(orjson.dumps(data))
Out[9]: {'lat': 31.24527019143944, 'lng': 121.48791951161944}

In [10]: json.loads(orjson.dumps(data))
Out[10]: {'lat': 31.245270191439438, 'lng': 121.48791951161945}

```

**Comment by ijl** [comment]
Ok, thanks for reporting. https://github.com/serde-rs/json/issues/128

**Comment by ijl** [comment]
https://github.com/ijl/orjson/commit/7a1434f4ca511c9e2ca1d2754a709d135f0137dc

**Comment by ijl** [comment]
Fixed in 2.0.5.
